src/benchmark/operators/sql_benchmark.cpp

```diff
@@ -5,6 +5,7 @@
 #include "SQLParser.h"
```

Comment:
(random line) @mrks 

We agreed we need the access to "AbstractLQPNode::node_expressions" better, currently it is two/three virtual functions that derived types need to override.

Idea: AbstractLQPNode, similar to `AbstractExpression::arguments` gets a member "std::vector<Expression> node_expressions" and just as "ArithmeticExpression::left_operand()" return "arguments[0]", "PredicateNode::predicate()" would return "node_expressions[0]". 

What do you think?

Reply:
Sounds reasonable. I can't think of any drawbacks - can you?

Reply:
Done